e2-studio-star-rx72n-firmware/libs/rx_comm_manager/src/rx_comm_manager.c

```
@@ -750,6 +815,144 @@ static rx_err_t internal_poll_spi(rx_comm_manager_t* mgr)
+{
+  if (mgr == nullptr || mgr->event_queue_count == 0) {
+    return;
+  }
+
+  rx_comm_event_entry_t* entry = &mgr->event_queue[mgr->event_queue_tail];
+  if (!entry->occupied) {
+    return;
+  }
+
+  /* Check backoff: skip if retry time hasn't been reached yet */
+  const uint32_t now_ms = internal_get_time_ms();
```

Clang (14.0.6)

[warning] variable 'now_ms' is not initialized

(cppcoreguidelines-init-variables)